docs/conf.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+
+from docutils.parsers.rst import roles
+from docutils import nodes
+import itertools
+import string
+import os
+import sphinx_rtd_theme
+import sys
+
+
+# -- General configuration ------------------------------------------------
+
+# Add the _extenions dir to the search path.
+sys.path.insert(0, os.path.abspath('.') + '/_extensions')
+sys.path.insert(0, os.path.abspath('.') + '/_extensions/haddock-autolink')
+
+extensions = ['sphinx.ext.intersphinx',
+              'sphinx.ext.ifconfig',
+              'haddock-autolink',
+              'hs-theme']
+
+templates_path = ['_templates']
+
+source_suffix = '.rst'
+
+master_doc = 'index'
+
+# General information about the project.
+project = u'unordered-containers'
+copyright = u'2020, Matt Renaud'
+author = u'Matt Renaud'
```

Comment:
Should this include the data of the original licenses, or is this just for the RTD?

Reply:
This is just for the docs, but I'm happy to change the copyright and author fields to "unordered-container maintainers". Wdyt?

Reply:
Works for me - that'd be great 😄

Reply:
Update.